coral-schema/src/test/java/com/linkedin/coral/schema/avro/TestUtils.java

```diff
@@ -184,10 +184,8 @@ private static void executeCreateTableWithPartitionQuery(String dbName, String t
   private static void executeCreateTableWithPartitionFieldSchemaQuery(String dbName, String tableName,
       String fieldSchema) {
     executeQuery("DROP TABLE IF EXISTS " + dbName + "." + tableName);
-    executeQuery("CREATE EXTERNAL TABLE " + tableName + " (" + fieldSchema + ") "
-        + "PARTITIONED BY (datepartition string) " + "ROW FORMAT SERDE 'org.apache.hadoop.hive.serde2.avro.AvroSerDe' "
-        + "STORED AS " + "INPUTFORMAT 'org.apache.hadoop.hive.ql.io.avro.AvroContainerInputFormat' "
-        + "OUTPUTFORMAT 'org.apache.hadoop.hive.ql.io.avro.AvroContainerOutputFormat'");
+    executeQuery(
```

Comment:
what does this simplification achieve? will the resulting schema format be avro / orc?

Reply:
Removing it because with Avro 1.10, we can't create a table just with [this kind of schema](https://github.com/linkedin/coral/blob/master/coral-schema/src/test/resources/base-complex-fieldschema)
the created table format is `inputFormat = "org.apache.hadoop.mapred.TextInputFormat",
outputFormat = "org.apache.hadoop.hive.ql.io.HiveIgnoreKeyTextOutputFormat"`

Reply:
(1) so all schema generation will go in the else case here https://github.com/linkedin/coral/blob/master/coral-schema/src/main/java/com/linkedin/coral/schema/avro/SchemaUtilities.java#L118  ?

(2) can you add this input format as an inline comment? it doesn't seem intuitive. 
(3) Also, how can we create and test avro format tables? or is that code path only tested on production views via integ tests?

Reply:
(1) (3) No, the other tables are created with Avro format by these 2 methods: [1](https://github.com/linkedin/coral/blob/d0a47081ced98c6aaa9db62eff5160189bad88b0/coral-schema/src/test/java/com/linkedin/coral/schema/avro/TestUtils.java#L166), [2](https://github.com/linkedin/coral/blob/d0a47081ced98c6aaa9db62eff5160189bad88b0/coral-schema/src/test/java/com/linkedin/coral/schema/avro/TestUtils.java#L175)
(2) Will add